### DISCUSSION TITLE
Valetudo Icon

### DISCUSSION THREAD

**Discussion by kusold** [selfhst/icons#85] [New Icon Request]
Cloud replacement for vacuum robots enabling local-only operation. It offers a WebUI to control your robot vacuum, and also can be controlled over MQTT via Home Assistant.

Website: https://valetudo.cloud
SVG: https://github.com/Hypfer/Valetudo/blob/master/docs/img/valetudo_logo_small.svg

**Comment by shollyethan** [comment]
Valetudo has been added - thanks!
